### ISSUE TITLE
Pictures being brought into the sequence are odd colors.

### ISSUE THREAD

**Issue by chrishhogan** [smeighan/xLights#3542]
When opening the lindsey sterling grinch sequence on the google drive up until 3 versions of xlights ago...the picture of the violin was brown...if you open the picture it is also brown...when seen in xlights, it is odd purple and pinkish color. I looked with a person from the zoom room and he said he had no clue there is a bug in the programming somewhere when bringing the pictures in.  The grinch also the hat was turned a hot pink color and he is highlighter green. 

![pink grinch](https://user-images.githubusercontent.com/117791057/200722367-50dc64b5-d517-4a6f-ad95-e6af3a1b746b.png)
![pink violin](https://user-images.githubusercontent.com/117791057/200722368-7ceb9de6-c8f2-4fc7-a473-c07f1d440e12.png)
![violin picture ](https://user-images.githubusercontent.com/117791057/200722369-92364547-3ffc-42f7-87c8-e11bccc32c0d.png)

NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)

**Expected behavior**
A clear and concise description of what you expected to happen and why.

**Screenshots**

**Versions (please complete the following information):**
 - OS: [e.g. Windows 10, Mac 10.14.5, etc...]
 - xLights version [e.g. 2019.37]

**Additional context**

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.

**Comment by derwin12** [comment]
Related to https://github.com/smeighan/xLights/issues/3524

**Comment by chrishhogan** [comment]
Ok...if I go back 4 versions of xlights it works perfect...just the last 3 that this has been happening...I showed a few people in zoom room and they said to just submit a bug form so that is what I did. The colors in real life show the same pink and purple as the xlights shows. What should I do? I ki d of need the new updates for the HPP on the new xlights updates but also need the colors to look ok or even close. 🤦

Sent from Yahoo Mail on Android 
 
  On Wed, Nov 9, 2022 at 11:34 AM, ***@***.***> wrote:   


Simple animation sequence with the pic in question.
PictureBug.zip

Not a fix but for a few folks with ones in zoom room we have suggest to not scale and use roto-Zoom with zoom and zoom quality. Only helpful for the one off fixes when someone points it out.
